README.md

```diff
@@ -1,11 +1,136 @@
+from torchmultimodal.transforms.bert_text_transform import BertTextTransform
+from torchmultimodal.transforms.flava_transform import FLAVAImageTransform
+
+# Define helper function for zero-shot prediction
+def predict(zero_shot_model, image, labels):
+    zero_shot_model.eval()
+    with torch.no_grad():
+        image = image_transform(img)["image"].unsqueeze(0)
+        texts = text_transform(labels)
+        _, image_features = zero_shot_model.encode_image(image, projection=True)
+        _, text_features = zero_shot_model.encode_text(texts, projection=True)
+        scores = image_features @ text_features.t()
+        probs = torch.nn.Softmax(dim=-1)(scores)
+        label = labels[torch.argmax(probs)]
+        print(
+            "Label probabilities: ",
+            {labels[i]: probs[:, i] for i in range(len(labels))},
+        )
+        print(f"Predicted label: {label}")
+
+
+image_transform = FLAVAImageTransform(is_train=False)
+text_transform = BertTextTransform()
+zero_shot_model = flava_model(pretrained=True)
+img = Image.open("my_image.jpg")  # point to your own image
+predict(zero_shot_model, img, ["dog", "cat", "house"])
+
+# Example output:
+# Label probabilities:  {'dog': tensor([0.80590]), 'cat': tensor([0.0971]), 'house': tensor([0.0970])}
+# Predicted label: dog
+  ```
+  </details>
+
+  <details>
+    <summary>MAE training example</summary>
+
+  ```python
+import torch
+from torch.utils.data import DataLoader
+from torchmultimodal.models.masked_auto_encoder.model import vit_l_16_image_mae
+from torchmultimodal.models.masked_auto_encoder.utils import (
+    CosineWithWarmupAndLRScaling,
+)
+from torchmultimodal.modules.losses.reconstruction_loss import ReconstructionLoss
+from torchmultimodal.transforms.mae_transform import ImagePretrainTransform
+
+mae_transform = ImagePretrainTransform()
+dataset = MyDatasetClass(transforms=mae_transform)  # you should define this
+dataloader = DataLoader(dataset, batch_size=8)
+
+# Instantiate model and loss
+mae_model = vit_l_16_image_mae()
+mae_loss = ReconstructionLoss()
+
+# Define optimizer and lr scheduler
+optimizer = torch.optim.AdamW(mae_model.parameters())
+lr_scheduler = CosineWithWarmupAndLRScaling(
+    optimizer, max_iters=1000, warmup_iters=100  # you should set these
+)
+
+# Train one epoch
+for batch in dataloader:
+    model_out = mae_model(batch["images"])
+    loss = mae_loss(model_out.decoder_pred, model_out.label_patches, model_out.mask)
+    loss.backward()
+    optimizer.step()
+    lr_scheduler.step()
+  ```
+  </details>
+
+## Code overview
+
+### [torchmultimodal/diffusion_labs](https://github.com/facebookresearch/multimodal/tree/main/torchmultimodal/diffusion_labs)
+diffusion_labs contains components for building diffusion models and end-to-end training of those models. For more details on these components, see [diffusion_labs/README.md](https://github.com/facebookresearch/multimodal/blob/main/torchmultimodal/diffusion_labs/README.md).
```

Comment:
It doesn't currently offer end to end training, and it's still an open question as to whether it will, so I'd remove that part.